src/widgets/buttons.ts

```diff
@@ -895,7 +927,7 @@ export function propertyTriage (kb: IndexedFormula): any {
   }
   possibleProperties.op = op
   possibleProperties.dp = dp
-  info(`propertyTriage: ${no} non-lit, ${nd} literal. ${nu} unknown.`)
+  info(`propertyTriage: ${no} non - lit, ${nd} literal.${nu} unknown.`)
```

Comment:
did you edit this on purpose?

Reply:
no, my save :(

Reply:
in this case it's not acceptable because it incorrectly changes the value of that string template.